hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/raft/StoreSnapshotFile.java

```diff
@@ -49,22 +48,27 @@
     private static final String SNAPSHOT_DIR = "ss";
     private static final String SNAPSHOT_ARCHIVE = "ss.zip";
 
-    public void save(BackendStore store, SnapshotWriter writer,
-                     Closure done, ExecutorService executor) {
+    private final RaftBackendStore[] stores;
+
+    public StoreSnapshotFile(RaftBackendStore[] stores) {
+        this.stores = stores;
+    }
+
+    public void save(SnapshotWriter writer, Closure done,
+                     ExecutorService executor) {
         String writerPath = writer.getPath();
         String snapshotPath = Paths.get(writerPath, SNAPSHOT_DIR).toString();
         try {
-            this.doSnapshotSave(store, snapshotPath)
-                .whenComplete((metaBuilder, throwable) -> {
-                if (throwable == null) {
+            this.doSnapshotSave(snapshotPath).whenComplete((metaBuilder, t) -> {
+                if (t == null) {
                     executor.execute(() -> compressSnapshot(writer, metaBuilder,
                                                             done));
                 } else {
                     LOG.error("Failed to save snapshot, path={}, files={}, {}.",
-                              writerPath, writer.listFiles(), throwable);
+                              writerPath, writer.listFiles(), t);
```

Comment:
"Failed to save snapshot, path={}, files={}", the exception stack will be printed